operator/v1/types_csi.go

```diff
@@ -0,0 +1,44 @@
+package v1
```

Comment:
please rename this file to something like `types_csi_snapshot.go`

Reply:
This has been renamed and updated.